### ISSUE TITLE
Add arm64 support

### ISSUE THREAD

**Issue by pnicolli** [plone/plone.docker#170]
Just tracking it here, it would be awesome to have this here even if [plone/plone-backend](plone/plone-backend) exists, since this is still used and is still the official library image.

I have no idea about where to go in order to add the arm64 build in the official library. Any pointers?
Also, I was looking at the github actions here. I assume the builds are pushed manually to docker hub, are they @avoinea ?

**Comment by avoinea** [comment]
@pnicolli Nope, they are not. See https://github.com/docker-library/official-images/blob/master/library/plone

And an update PR https://github.com/docker-library/official-images/pull/11795/files

And the ARM architectures where remove by:
- https://github.com/docker-library/official-images/pull/11303/files
- https://github.com/docker-library/official-images/pull/11291/files

**Comment by avoinea** [comment]
@pnicolli If you have the possibility to build and test the images on ARM architectures we can re-include them when we do the PR to official-images for #168 and #169

**Comment by avoinea** [comment]
@pnicolli Would be nice if we would be able to test on ARM via github actions :exploding_head:

**Comment by pnicolli** [comment]
> @pnicolli Nope, they are not. See https://github.com/docker-library/official-images/blob/master/library/plone
> 
> And an update PR https://github.com/docker-library/official-images/pull/11795/files

Yeah sorry, my question was about the unofficial images [plone/plone](https://hub.docker.com/r/plone/plone), but I just realized they are not kept up to date, which is also fine.

> @pnicolli If you have the possibility to build and test the images on ARM architectures we can re-include them when we do the PR to official-images for https://github.com/plone/plone.docker/issues/168 and https://github.com/plone/plone.docker/issues/169

I will let you know if I can look into this, I recently moved to a newer arm powered laptop.

> @pnicolli Would be nice if we would be able to test on ARM via github actions 🤯

I believe this is doable, I will try

**Comment by ericof** [comment]
Closing as this repository is kept only for historical reasons
